src/modules/logger/logger.cpp

```diff
@@ -1375,16 +1376,8 @@ void Logger::print_load_callback(void *user)
 
 void Logger::initialize_load_output(PrintLoadReason reason)
 {
-	perf_callback_data_t callback_data;
-	callback_data.logger = this;
-	callback_data.counter = 0;
-	callback_data.buffer = nullptr;
-	char buffer[140];
-	hrt_abstime curr_time = hrt_absolute_time();
-	init_print_load_s(curr_time, &_load);
-	// this will not yet print anything
-	print_load_buffer(curr_time, buffer, sizeof(buffer), print_load_callback, &callback_data, &_load);
-	_next_load_print = curr_time + 1000000;
+	init_print_load(&_load);
+	_next_load_print = hrt_absolute_time() + 2_s;
```

Comment:
Why is this 2s now?

Reply:
When you start it zeros any existing times and the scheduling notes begin to accumulate runtime again. From watching `top` it generally seemed to take more than 1 second to actually catch everything and settle to numbers consistent with running top now.

Reply:
Is 1.5s enough? I'd like to keep it as low as we can.

Reply:
We'll have to experiment a bit. Running `top` seemed to need >= 2 seconds to settle, but now that I think about it, the 1 second loop in top_main might be a contributing factor.

Reply:
I've left it at 1 second, but initialized slightly earlier where possible (preflight).